packages/styles/src/components/header/index.scss

```diff
@@ -10,7 +10,16 @@ tokens.$default-map: elements.$post-link;
 
 [data-post-scroll-locked] {
   overflow: hidden !important;
-  transform: translate3d(0, 0, 0) !important;
+
+  &:not(body) {
+    transform: translate3d(0, 0, 0) !important;
```

Comment:
Still not sure about this. I think we should refactor the header to not rely on this to be there. What are our options here?